### PR TITLE
network: plugins installation documentation

### PR DESCRIPTION
**Pull request by phoracek**

### CODE REVIEW

**Comment by phoracek** [comment]
@stu-gott could you please take a look?

File: installation/README.md [hunk 1]
```diff
@@ -132,6 +132,16 @@ The [kubevirt-ansible](https://github.com/kubevirt/kubevirt-ansible) project pro
...
 
+## Installing network plugins (optional)
+
+KubeVirt alone does not bring any additional network plugins, it just allows
+user to utilize them. If you want to attach your VMs to multiple networks
+(Multus CNI) or have full control over L2 (OVS CNI), you need to deploy
+respective network plugins. For more information, refer to [OVS CNI installation
+guide](https://github.com/kubevirt/ovs-cni/blob/master/docs/deployment-on-arbitrary-cluster.md).
+
+> Note: Ansible playbooks install these plugins by default.
```
Comment (fabiand) [inline installation/README.md]: @petrkotas please note and link which ansible playbook would do this.

**Comment by phoracek** [comment]
@fabiand done

File: installation/README.md [hunk 2]
```diff
@@ -132,6 +132,16 @@ The [kubevirt-ansible](https://github.com/kubevirt/kubevirt-ansible) project pro
 
 See the [Developer Getting Started Guide](https://github.com/kubevirt/kubevirt/blob/master/docs/getting-started.md) to understand how to build and deploy KubeVirt from source.
 
+## Installing network plugins (optional)
+
+KubeVirt alone does not bring any additional network plugins, it just allows
+user to utilize them. If you want to attach your VMs to multiple networks
+(Multus CNI) or have full control over L2 (OVS CNI), you need to deploy
+respective network plugins. For more information, refer to [OVS CNI installation
+guide](https://github.com/kubevirt/ovs-cni/blob/master/docs/deployment-on-arbitrary-cluster.md).
```
Comment (petrkotas) [inline installation/README.md]: When I read install guide, it helps to have every bit of information at hands. The less I am required to jump, the better I can understand the guide.
Can you please update the list of plugins here, better yet provide simple example?

I have looked at the kubevirt ansible and it seems it is only installed as [part of multus](https://github.com/kubevirt/kubevirt-ansible/search?q=ovs-cni&unscoped_q=ovs-cni). If I want the plugins and do not want multus, I would not have and option to install it.

Comment (fabiand) [reply installation/README.md]: Was there any conclusion on this item?
As I now see that you approved the PR, but this q was not answered here.

Comment (petrkotas) [reply installation/README.md]: I talked to @phoracek and we resolved it. It is not required here.

**Pull request merged**